src/linker/utilities/slurm_utils.py

```diff
@@ -51,6 +53,7 @@ def launch_slurm_job(
     ]
     for filepath in input_data:
         jt_args.extend(("--input-data", str(filepath)))
+    jt_args.extend(("--config", str(json.dumps(config)))) if config else None
```

Comment:
I'm not used to single-line if being used in this way, but maybe that's just me.

I'm assuming that DRMAA is adequately quoting this? What happens if you try to pass an environment variable with a quote in it?

Reply:
Good question, though I have no idea right now. Will mess around w/ it

Reply:
It seems to parse fine? I'll leave it for now since we don't have any cases of env vars with quotes but I think it should work
![image](https://github.com/ihmeuw/linker/assets/23350991/f8f68dc3-ff38-45f4-bebf-04c56bc250c6)

Reply:
I think I agree with Zeb that doing this is easier to read:
```suggestion
    if config:
        jt_args.extend(("--config", str(json.dumps(config))))
```